src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -127,6 +135,30 @@ public void setPerson(Person target, Person editedPerson) {
         addressBook.setPerson(target, editedPerson);
     }
 
+    @Override
+    public boolean hasTutorial(Tutorial tutorial) {
+        requireNonNull(tutorial);
+        return addressBook.hasTutorial(tutorial);
+    }
+
+    @Override
+    public void deleteTutorial(Tutorial target) {
+        addressBook.removeTutorial(target);
+    }
+
+    @Override
+    public void addTutorial(Tutorial tutorial) {
+        addressBook.addTutorial(tutorial);
+        updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
```

Comment:
just wondering, why do you need to update the filteredPersons list if there is no change to any person in the address book?

Reply:
Thanks, I changed it to updateFilteredTutorialList(SHOW_ALL_TUTORIALS).

For the case where the user already has the tutorialList displayed (filtered or not), and adding a new class should immediately show all the classes in the app.